_data/header_menu.yml

```diff
@@ -2,6 +2,8 @@
   link: https://blog.rustfest.eu/
 - title: Tickets
   link: /tickets/
+- title: Pre-/Post Events
+  link: /pre_post_events/
```

Comment:
this breaks the site layout. Do we need it in the header?

Reply:
plus it's an orphan page. Should we create a stub first?

Reply:
Or we could include more links in the menu and then it is a feature? :innocent:

Reply:
It's already suboptimal on smaller screens.
On mobile it essentially takes a whole screen with just the menu.

I'd say let's not make it worse right now here, this can be tackled independently. I'd rather have this land without a frontpage-link